index.html

```diff
@@ -68,6 +67,123 @@ <h2 id="support-text">I'm a Software Developer</h2>
     <img id="main-img" src="assets/illustration.svg" alt="">
   </section>
 
+  <section id="recent-works">
+    <div class="container">
+      <div class="section-title">
+        <h1 id="recent-works-title">My Recent Works</h1>
+      </div>
+      <div class="line"></div>
+      <img id="recent-works-img" src="assets/img-placeholder.png" alt="">
+      <div id="post-stories">
+        <h1>Multi-Post Stories</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="row-container">
+          <div class="text-box">
+            <p>CSS</p>
+          </div>
+          <div class="text-box">
+            <p>HTML</p>
+          </div>
+          <div class="text-box">
+            <p>Bootstrap</p>
+          </div>
+          <div class="text-box">
+            <p>Ruby</p>
+          </div>
+        </div>
+        <div class="orange-button">
+          <a class="btn-text">See project</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="work1">
+    <div class="container">
+      <div id="post-stories">
+        <h1>Profesional Art Printing Data</h1>
```

Comment:
- [ ] As seen from the Figma design, please kindly add an image for each project.
- [ ] As seen from the Figma design, please ensure that your header and paragraph are color white

| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="283" alt="image" src="https://user-images.githubusercontent.com/52934128/227751547-d32a9f8c-c54b-460e-8846-63b4a4841ea1.png"> | <img width="246" alt="image" src="https://user-images.githubusercontent.com/52934128/227751522-ffec7b09-f899-426c-9059-05edcf6a03a4.png"> |

Reply:
I fixed the color of the header and paragraph but I'm having trouble displaying the background image, as you can see in my project it only shows a background going from white to gray. Could please provide further explanation on how I could add the background img like in template, since there are a lot of layers on the template I don't understand really well what exactly is going on.